src/main/java/duke/command/AddTodoCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.common.Messages;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.ui.TextUi;
+
+public class AddTodoCommand extends Command {
```

Comment:
Perhaps follow a consistent style in naming classes extending `Command` class?
```suggestion
public class TodoCommand extends Command {
```